server/services/image-optimizer-service.ts

```diff
@@ -134,11 +136,6 @@ function sharpAddResizeSettings(
   size: ImageSize,
   factor: number
 ): Sharp {
-  if (!size.width && !size.height) {
-    throw new InvalidParametersError(
-      "Either width or height must be specified."
-    );
-  }
```

Comment:
Simply removing the check will not work as expected. If neither width nor height are defined, no additional resolutions will be created, which would be a bug. Instead, the size of the original image should be used.